slack_bolt/cli/get_hooks.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import json
+from .protocol import Protocol, MessageBoundaryProtocol, DefaultProtocol, protocol_factory
+from slack_bolt.cli import get_manifest, start
+
+PROTOCOL: Protocol
+
+hooks_payload = {
+    "hooks": {
+        "get-manifest": f"python -m {get_manifest.__name__}",
+        "start": f"python -m {start.__name__}",
```

Comment:
Are there any concerns around using the `python` runtime? For example, my macOS system doesn't have a `python` binary installed and instead uses `python3`.
<img width="329" alt="image" src="https://github.com/slackapi/bolt-python/assets/21328/28b180cc-d50b-4616-9288-05a8a45d61a7">

Reply:
Using `python3` instead of `python` seem to be the safer option